### PR TITLE
Introduce 'callback_yaml' environment variable

### PR DESCRIPTION
**Pull request by zinid**
Refer to updated README for the explanation

### CODE REVIEW

File: src/conf_env.erl [hunk 1]
```diff
@@ -0,0 +1,115 @@
+%%%-------------------------------------------------------------------
+%%% @author Evgeny Khramtsov <ekhramtsov@bdt.group>
+%%% @copyright (C) 2020, Big Data Technology
```
Comment (arentrue) [inline src/conf_env.erl]: I would use the same header as in other modules in the project. This proprietary copyright looks alien in Apache project)

Comment (arentrue) [reply src/conf_env.erl]: Otherwise PR looks fine)

Comment (zinid) [reply src/conf_env.erl]: Thanks a ton for spotting this!

Comment (zinid) [reply src/conf_env.erl]: Fixed.

**Pull request merged**